042-remove-bridge-amqp-support.md

```diff
@@ -0,0 +1,46 @@
+# Remove AMQP 1.0 support from the Strimzi bridge
+
+This proposal is about removing the current support for the AMQP 1.0 protocol from the Strimzi bridge, leaving just the HTTP support.
+
+## Current situation
+
+Currently the Strimzi bridge provides support for the AMQP 1.0 protocol other than HTTP.
+The AMQP 1.0 protocol support is provided by using the [Vert.x Proton](https://github.com/vert-x3/vertx-proton) component for handling the communication on the wire.
+The current implementation doesn't follow any specific standard, like the new available [Event Stream Extensions for AMQP Version 1.0](https://docs.oasis-open.org/amqp/event-streams/v1.0/csd01/event-streams-v1.0-csd01.html).
```

Comment:
Can to clarify what exactly this means? It's confusing because a previous sentence says "provides support for the AMQP 1.0 protocol".

Reply:
The point I was trying to make is that the current implementation is based on my own design, in terms of AMQP message properties usage, addresses patterns and so on (you can find my "own" specification here https://github.com/strimzi/strimzi-kafka-bridge/blob/main/amqp/AMQP_intro.md) It's different from having the OASIS standard defining them and you have to follow that specification.
How do you think I can improve this sentence to make it clearer?

"The current implementation comes from a custom design in terms of usage of AMQP 1.0 message properties, addresses and so on but not following any specific standard, like ...."

Any better suggestion?

Reply:
Yeah, that makes it much clearer what you mean. SGTM.